Feature/storage scripts

GustavoRSSilva opened the pull request:
New script to add a advertisement address to the advertisement address

DioPires commented on scripts/advertisement-storage.js:
```diff
@@ -0,0 +1,44 @@
+const AdvertisementStorage = artifacts.require("./AdvertisementStorage.sol");
```
#### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

DioPires replied:
#### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

DioPires replied:
#### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

DioPires replied:
#### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

DioPires replied:
#### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

DioPires replied:
#### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

DioPires replied:
#### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

GustavoRSSilva commented:
added the documentation:

https://github.com/AppStoreFoundation/asf-contracts/wiki/AdvertisementStorage-contract

Added the advertisement contract to the AdvertisementStorage in the ropsten network

![screenshot from 2018-06-05 11-19-06](https://user-images.githubusercontent.com/8384988/40971150-938a2f6a-68b4-11e8-97e0-36591092a4bf.png)

tcrosado commented:
:tada: This PR is included in version 1.0.0 :tada:

The release is available on [GitHub release](https://github.com/AppStoreFoundation/asf-contracts/releases/tag/v1.0.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
